spec/plans/ansible.fmf

```diff
@@ -0,0 +1,45 @@
+summary: Define Ansible general settings for the run
+
+story:
+    As a user, I want to configure Ansible related settings for the test run,
+    for example, the ability to define the structure of the generated inventory
+    using a customizable layout template.
+
+description: |
+    When running tests, a static inventory file is generated from
+    provision metadata. The ``ansible.inventory.layout`` key allows
+    users to specify a YAML file defining the desired inventory group
```

Comment:
Will the file be treated as a template? IIRC, that was the idea, I would mention it here right from the beginning.

Reply:
Output file is static. Input layout can be just the template layout or the inventory in it's final state

In principle, dynamic inventory is not supported. That's why I specified static